### ISSUE TITLE
Increase horizontal label padding

### ISSUE THREAD

**Issue by winkelsdorf** [react-navigation/react-navigation-tabs#113]
For iOS horizontal label padding the default is given in `labelBeside`:
https://github.com/react-navigation/react-navigation-tabs/blob/caa50fe78207be8c8a33d3ef3ba156ff84e6f544/src/views/BottomTabBar.js#L464

Example with `react-navigation` defaults (except for tint):
<img width="166" alt="Screen Shot 2019-04-03 at 17 58 47" src="https://user-images.githubusercontent.com/1413291/55493996-3f346100-563a-11e9-9070-3f250e36a8f0.png">

Example from Apple native padding:
<img width="808" alt="Screen Shot 2019-04-03 at 17 57 03" src="https://user-images.githubusercontent.com/1413291/55493983-39d71680-563a-11e9-9e9b-a5b584c3be35.png">

In comparison I would say, the `labelBeside` needs a bit more space.

Something like `25` looks right:
<img width="150" alt="Screen Shot 2019-04-03 at 18 01 48" src="https://user-images.githubusercontent.com/1413291/55494198-9c301700-563a-11e9-8322-fb5119effd80.png">

**Comment by brentvatne** [comment]
re: the uikit default padding, it appears that the icon is also significantly larger in the react-navigation screenshot in your app. you may want to just shrink it when in horizontal mode, you can do this like so:

```js
tabBarIcon: ({ horizontal, tintColor }) => <Icon name="whatever" size={horizontal? 20 : 25} color={tintColor} />
```

**Comment by winkelsdorf** [comment]
Good catch @brentvatne, I agree they do look a bit large! Sadly applying conditional size still does not give the expected result. 

Btw icons should work with 25 pt with iPads [^1], don't they? iPad's TabBar is behaving differently, as compact != horizontal there. On iPad the `UITabBar` is never compact [^2], no matter what trait is used.

But that's already taken care of here:
https://github.com/react-navigation/react-navigation-tabs/blob/caa50fe78207be8c8a33d3ef3ba156ff84e6f544/src/views/BottomTabBar.js#L318

I'm using images (size @1 is 25px up to 75px @3) with the following `navigationOptions` for each of the tabs (including your suggested change now):
```js
tabBarIcon: ({ focused, tintColor, horizontal }) => (
    focused
    ? <Image size={horizontal ? 20 : 25} source={Icons.productFilled} style={{ tintColor: tintColor }}/>
    : <Image size={horizontal ? 20 : 25} source={Icons.product} style={{ tintColor: tintColor }}/>
),
```

Still similar:
<img width="243" alt="Screen Shot 2019-04-04 at 00 32 18" src="https://user-images.githubusercontent.com/1413291/55518382-15982b80-5674-11e9-9494-a688f8097ca1.png">

[^1]: https://developer.apple.com/design/human-interface-guidelines/ios/icons-and-images/custom-icons/ under "Tab Bar Icon Size"

[^2]: https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/adaptivity-and-layout/ under "Device Size Classes"

**Comment by winkelsdorf** [comment]
Verified the icon size not to be the source of the problem. Image is already fixed to 25 (px) in height and width with `resizeMode: "cover"`. Using a very large icon (> 600px) I was able to verify the limits to be working correctly.

So it is actually the padding for `labelBeside`.

<img width="1148" alt="Screen Shot 2019-04-10 at 11 07 49" src="https://user-images.githubusercontent.com/1413291/55866562-e6f0e800-5b80-11e9-90e6-a00aa7381744.png">
